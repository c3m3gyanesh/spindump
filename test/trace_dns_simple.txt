Simple DNS example, just one A query and one result.
