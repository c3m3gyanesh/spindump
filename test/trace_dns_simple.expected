DNS 10.0.1.36 <-> 193.234.218.146 65456:53 at 1571841388453497 new starting packets 0 0 bytes 0 0
DNS 10.0.1.36 <-> 193.234.218.146 65456:53 at 1571841388603362 measurement starting right 149865 packets 1 0 bytes 58 0 bandwidth 58 0
